### ISSUE TITLE
what should clear when chain and combo garbage touching

### ISSUE THREAD

**Issue by omenking** [omenking/swap-n-pop#291]
Lets say we have a stack of garbage touching.
For each garbage chain group only one line will clear per garbage chain group, always the bottom row.
All touching combo will clear as long as its not separated by a shock garbage row.

**Comment by conormackeigan** [comment]
Correct, but it's probably better to think of it as "the same garbage type is touching", regardless of shock as both shock and regular panels can prevent garbage from touching

**Comment by conormackeigan** [comment]
"The same garbage type" meaning either combo&chain garbage vs shock garbage
